### DISCUSSION TITLE
Mlife binary file error

### DISCUSSION THREAD

**Discussion by windexplorer21** [OpenFAST/openfast#2468] [Q&A]
Hello everyone,

I'm trying to run the Mlife input example file 'Test01.mlif' with the 'Test1' FAST output file located in the 'Data' folder. While the text output file 'Test1.out' gives the correct results, switching to the binary output file 'Test1.outb' causes MATLAB to display this error:
```
Error using process_mlife_inputs
  For fatigue channel #1, the channel number must be between 1 and
  7 (inclusive).
```
Both the text and binary output files have 113 channels. Could anyone help me understand what might be causing this problem?

Thank you in advance for your time and assistance!

Best regards,
windexplorer21

**Comment by jjonkman** [comment]
Dear @windexplorer21,

Have you changed `FileFormat` in MLife from 1 (ascii) to 2 (binary)?

Best regards,

**Reply by windexplorer21** [reply]
Dear @jjonkman 
Thank you for your response. Yes, that is the only change I made, aside from the file name of course.

@windexplorer21

**Reply by jjonkman** [reply]
Dear @windexplorer21,

Does MLife Test26 run for you?  Test26 is identical to Test27, except that Test26 uses the binary versions of the asci output files used by Test27.

Best regards,
